### DISCUSSION TITLE
Uncaught SyntaxError: Unexpected token '<'

### DISCUSSION THREAD

**Discussion by louis1999** [jeffreylanters/react-unity-webgl#348] [Help Wanted!] [answered]
Hello,

My errors are : 
- Uncaught SyntaxError: Unexpected token '<'
- A problem occurred while mounting Error: Unable to invoke Unity Loader, loaded incorrectly.

I know that these errors are common but the weird thing is depending where is use my component, sometime i have the error and sometimes not.

I have my webgl project correctly set inside the public folder like follow
>public
    >FallingGame
        >Build
            >FallingGame.data
            >FallingGame.framework.js
            >FallingGame.loader.js
            >FallingGame.wasm

In my js file, i create my unity context like follow : 
  const unityContext = new UnityContext({
    loaderUrl: "./FallingGame/Build/FallingGame.loader.js",
    frameworkUrl: "./FallingGame/Build/FallingGame.framework.js",
    dataUrl: "./FallingGame/Build/FallingGame.data",
    codeUrl: "./FallingGame/Build/FallingGame.wasm",
  });

And i display it like that : 
<Unity
          unityContext={unityContext}
/>

The errors DO NOT OCCUR when I use the component in a function or class that has a short path (either the root or juste one field like /home or /user)
The errors OCCUR when I use the component in a function or class that has a longer path that one field (like /user/account our /course/game)

This is a really weird problem, i use exactly the same code but it doesn't work in every file, the length of the url path is the only thing I found that differ in the 2 different cases

Any hint i can get to resolve that ?

Thanks in advance 

Louis

**Comment by louis1999** [comment]
<img width="517" alt="image" src="https://user-images.githubusercontent.com/33379335/158371791-04623953-2c19-4b8f-97a7-80a742d56bc8.png">

**Comment by louis1999** [comment, marked as answer]
I HAVE FOUND THE ISSUE
I am so sorry, I was looking for 2 days after a solution
And with writing that question that helped me defining the problem.
The url path was indeed the problem
if the path is root or /field, that works correctly
if the path is longer, like /user/account, you have to add ../
Like that
const unityContext = new UnityContext({
  loaderUrl: "./../FallingGame/Build/FallingGame.loader.js",
  frameworkUrl: "./../FallingGame/Build/FallingGame.framework.js",
  dataUrl: "./../FallingGame/Build/FallingGame.data",
  codeUrl: "./../FallingGame/Build/FallingGame.wasm",
});

I hope that can help you

**Reply by ZDev-9** [reply]
YOOOOO You legit just saved me like a day and a half of pulling my hair out. 
Thank you so much!
